structural-variantcalling.wdl

```diff
@@ -0,0 +1,101 @@
+version 1.0
+
+import "tasks/common.wdl" as common
+import "tasks/delly.wdl" as delly
+import "tasks/manta.wdl" as manta
+import "tasks/picard.wdl" as picard
+import "tasks/bcftools.wdl" as bcftools
+import "tasks/survivor.wdl" as survivor
+import "tasks/clever.wdl" as clever
+import "tasks/bwa.wdl" as bwa
+import "tasks/samtools.wdl" as samtools
+
+workflow SVcalling {
+    input {
+        File bamFile
+        File bamIndex
+        File referenceFasta
+        File referenceFastaFai
+        File referenceFastaDict
+        BwaIndex bwaIndex
+        String sample
+        String outputDir = "."
+    }
```

Comment:
There is no dockerImages input. Did the tasks already use docker images? If so they should be configurable. You can look at one of the other pipelines for an example on how to handle this.

Reply:
I already added the  dockerImagesFile